kale/prepdata/supergraph_construct.py

```diff
@@ -0,0 +1,250 @@
+import logging
+from typing import Dict, List
+
+import networkx as nx
+import torch
+
+
+class SuperVertex(object):
+    r"""
+    The supervertex structure in GripNet. Each supervertex is a subgraph containing nodes with the same category
+    and at least keep semantically-coherent. Supervertices can be homogeneous or heterogeneous.
```

Comment:
nodes with the same category and at least keep semantically-coherent --> nodes of the same category
    that are semantically-coherent.

Reply:
Changed in [5c193a3](https://github.com/pykale/pykale/pull/338/commits/5c193a39c6e3d76aa9249ae5ff9546a17d6c60a2).